File: 卡码/KamaCoder/C++/test04.cpp
//4. A+B问题IV
#include <iostream>
using namespace std;

int main(){
	int n;
	while(scanf("%d",&n)!=EOF){
		if(n==0) break;
		int sum=0;
		for(int i=0;i<n;i++){
			int tmp;
			scanf("%d",&tmp);
			sum+=tmp;
		}
		printf("%d\n",sum);
	}
	return 0;
}

File: 卡码/KamaCoder/Go/test04.go
//4. A+B问题IV
package main

import (
	"fmt"
)

func main(){
	for {
		var n int
		fmt.Scan(&n)
		if n==0 {
			return
		}
		var sum=0
		for i:=0;i<n;i++ {
			var tmp int
			fmt.Scan(&tmp)
			sum+=tmp
		}
		fmt.Println(sum)
	}
}
